src/_pytest/assertion/util.py

```diff
@@ -228,10 +230,32 @@ def _diff_text(left: str, right: str, verbose: int = 0) -> List[str]:
         explanation += ["Strings contain only whitespace, escaping them using repr()"]
     # "right" is the expected base against which we compare "left",
     # see https://github.com/pytest-dev/pytest/issues/3333
-    explanation += [
-        line.strip("\n")
-        for line in ndiff(right.splitlines(keepends), left.splitlines(keepends))
-    ]
+
+    from difflib import SequenceMatcher
```

Comment:
Should move to the top of the function (next to existing `difflib` import).

Reply:
Done.